tests/ui/write-fmt-errors.rs

```diff
@@ -1,9 +1,11 @@
 //@ run-pass
+//@ needs-unwind
```

Comment:
can you make it a should_panic unit test?

Reply:
Then I'd have to split it in 3 tests -- the parts that shouldn't panic, and the two panics.
I'd rather not do that.